ts/components/conversation/composition/CharacterCount.tsx

```diff
@@ -31,10 +32,10 @@ const StyledRemainingNumber = styled.span<{ pastLimit: boolean }>`
 `;
 
 export function CharacterCount({ count }: CharacterCountProps) {
-  const alwaysShowFlag = getFeatureFlag('alwaysShowRemainingChars');
-  // const dispatch = useDispatch();
-  const isProAvailable = getFeatureFlag('proAvailable');
-  const mockHasPro = getFeatureFlag('mockUserHasPro');
+  const alwaysShowFlag = getFeatureFlag('useAlwaysShowRemainingChars');
+  const dispatch = useDispatch();
+  const isProAvailable = getFeatureFlag('useProAvailable');
+  const mockHasPro = getFeatureFlag('useMockUserHasPro');
```

Comment:
should you make those hooks? i.e. start with `use`. I know they are not hooks, but would be nicer I think that way